src/main/resources/static/app.css

```
@@ -708,6 +714,17 @@ textarea.input { padding: 12px; resize: vertical; }
   70% { transform: translateY(0); }
 }
 
+@keyframes panelGlow {
```

Stylelint (17.9.0)

[error] Expected keyframe name "panelGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)